packages/front-end/components/Settings/ApiKeysModal.tsx

```diff
@@ -63,6 +63,18 @@ const ApiKeysModal: FC<{
       submit={onSubmit}
       cta="Create"
     >
+      {!secret && projects.length > 0 && (
+        <SelectField
```

Comment:
Consider `projects[]` scoping (\<MultiSelect\>) to avoid needing multiple SDK implementations on the client side.